### ISSUE TITLE
Docker support environment variables?

### ISSUE THREAD

**Issue by TemnikovY** [Shopify/toxiproxy#236]
Hello there!

Can I specify proxies configuration using environment variables passed by command line when I call
"docker run -e "<proxy configuration>" ..." ?

**Comment by jpittis** [comment]
I don't believe that's currently possible. Right now you need to start the container and then hit the HTTP API with "create proxy commands".

I feel like the idea of an "initial state" specified as a file by command line flag might be useful. (Has this been discussed before?)

**Comment by zeisss** [comment]
Afaik `toxiproxy` has a `--config-file` option which takes a list of `Proxy` objects as defined for the HTTP API.
